code/game/objects/structures/crates_lockers/crates/graves.dm

```diff
@@ -0,0 +1,116 @@
+	var/first_open = FALSE
+
+/obj/structure/closet/crate/grave/attackby(obj/item/W, mob/user, params)
+	.=..()
+	if(istype(W, /obj/item/screwdriver))
+		if(!user.is_literate())
+			to_chat(user, "<span class='notice'>You scratch illegibly on [src]!</span>")
+			return
+		var/t = stripped_input(user, "What would you like the inscription to be?", name, null, 53)
+		if(user.get_active_held_item() != W)
+			return
+		if(!user.canUseTopic(src, BE_CLOSE))
+			return
+		if(t)
+			desc = "[t]"
+		return
+
+/obj/structure/closet/crate/grave/open(mob/living/user, obj/item/S, force = FALSE)
+	if(!opened)
+		to_chat(user, "<span class='notice'>The ground here is too hard to dig up with your bare hands. You'll need a shovel.</span>")
+	else
+		to_chat(user, "<span class='notice'>The grave has already been dug up.</span>")
+
+/obj/structure/closet/crate/grave/tool_interact(obj/item/S, mob/living/carbon/user)
+	if(user.a_intent == INTENT_HELP) //checks to attempt to dig the grave, must be done on help intent only.
+		if(!opened)
+			if(S.tool_behaviour == cutting_tool)
+				to_chat(user, "<span class='notice'>You start start to dig open \the [src]  with \the [S]...</span>")
+				if (do_after(user,20, target = src))
+					opened = TRUE
+					locked = TRUE
+					dump_contents()
+					update_appearance()
+					SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "graverobbing", /datum/mood_event/graverobbing)
+					if(lead_tomb == TRUE && first_open == TRUE)
+						user.gain_trauma(/datum/brain_trauma/magic/stalker)
+						to_chat(user, "<span class='boldwarning'>Oh no, no no no, THEY'RE EVERYWHERE! EVERY ONE OF THEM IS EVERYWHERE!</span>")
+						first_open = FALSE
+					return 1
+				return 1
+			else
+				to_chat(user, "<span class='notice'>You can't dig up a grave with \the [S.name].</span>")
+				return 1
+		else
+			to_chat(user, "<span class='notice'>The grave has already been dug up.</span>")
+			return 1
+
+	else if((user.a_intent != INTENT_HELP) && opened) //checks to attempt to remove the grave entirely.
+		if(S.tool_behaviour == cutting_tool)
+			to_chat(user, "<span class='notice'>You start to remove \the [src]  with \the [S].</span>")
+			if (do_after(user,15, target = src))
+				to_chat(user, "<span class='notice'>You remove \the [src]  completely.</span>")
+				SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "graverobbing", /datum/mood_event/graverobbing)
+				deconstruct(TRUE)
+				return 1
+	return
+
+/obj/structure/closet/crate/grave/bust_open()
+	..()
+	opened = TRUE
+	update_appearance()
+	dump_contents()
+	return
+
+/obj/structure/closet/crate/grave/stone
+	name = "burial mound"
+	desc = "A marked patch of soil, adorned with a sandstone slab"
+	icon_state = "grave_lead"
+
+/obj/structure/closet/crate/grave/loot
+	name = "burial mound"
+	desc = "A marked patch of soil, showing signs of a burial long ago. You wouldn't disturb a grave... right?"
+	opened = FALSE
+
+/obj/structure/closet/crate/grave/loot/PopulateContents()  //GRAVEROBBING IS NOW A FEATURE
```

Comment:
Also feel free to update this loot list with new stuff if you want. These graves are used pretty commonly on a few maps and it'd be neat to see them contain some new stuff.

Reply:
will do tomorrow morning.